/**:
  ros__parameters:
    cloud_ip: "*"
    cloud_manage_port: 25565
    cloud_sub_port: 25567
    cloud_pub_port: 25568
    cloud_req_port: 25569
    cloud_rep_port: 25570

    sub_list: ["cmd_vel", "plan", "footprint", "bt_elev_feedback", "bt_elev_result"]
    # pub_list: ["scan", "tf", "tf_static", "odom", "imu", "clock", "bt_elev"]
    # pub_list: ["scan"]
    # sub_list: ["cmd_vel"]
    tf_list: ["map_odom"]
    srv_client_list: ["load_map"]
    srv_server_list: ["call_elevator", "get_called_elevator", "get_elevator_information", "is_door_opened", "request_door_open", "select_elevator_floor"]
